docs/conf.py

```diff
@@ -22,20 +23,74 @@
 import tmt.utils
 from tmt.utils import Path
 
-try:
-    # use bootstrap theme if user has it installed
-    import sphinx_bootstrap_theme
-    HTML_THEME = 'bootstrap'
-    html_theme_path = [sphinx_bootstrap_theme.get_html_theme_path()]
-except ImportError:
+_POSSIBLE_THEMES: List[Tuple[Optional[str], str]] = [
+    # Fall back to sphinx_rtd_theme if available
+    ('sphinx_rtd_theme', 'sphinx_rtd_theme'),
+    # The default theme
+    (None, 'default')
+    ]
+
+# NOTE: this one is defined somewhere below, among original Sphinx config fields,
+# but we need it as early as possible to be set when loading themes.
+# Add any paths that contain custom themes here, relative to this directory.
+html_theme_path = []
+
+
+def _load_theme(
+        theme_package_name: str,
+        theme_name: str) -> bool:
     try:
-        # fall back to sphinx_rtd_theme if available
-        import sphinx_rtd_theme
-        HTML_THEME = 'sphinx_rtd_theme'
-        html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-    except ImportError:
-        # and fall back to 'default' if neither of those are available
-        HTML_THEME = 'default'
+        theme_package = importlib.import_module(theme_package_name)
+
+    except ModuleNotFoundError:
+        return False
+
+    global HTML_THEME, html_theme_path
+
+    if hasattr(theme_package, 'get_html_theme_path'):
+        global html_theme_path
+
+        HTML_THEME = theme_name
```

Comment:
Duplicate of line 61?

Reply:
There's a `return True` later on. But yeah this could be moved outside of it. Also could the line-spacing be compressed a bit here?

Reply:
The line dropped in 88497cc9.

WRT the empty lines - unless I'm forced to do so, I'm going to keep them. Empty lines help me follow the logical steps of the code.

Reply:
I guess a wrong lines has been removed in 88497cc? I think `html_theme_path` needs to be marked with `global` and `HTML_THEME = theme_name` is the duplicate.

Reply:
Hopefully sorted out in 18a77322